packages/web/test/src/lib/myalgowallet-mode.ts

```diff
@@ -0,0 +1,111 @@
+import MyAlgoConnect from "@randlabs/myalgo-connect";
+import algosdk, { Account, Transaction } from "algosdk";
+import assert from "assert";
+
+import { MyAlgoWalletSession, types } from "../../../src";
+import { algoexplorerAlgod, getSuggestedParams } from "../../../src/lib/api";
+import { HttpNetworkConfig } from "../../../src/types";
+import MyAlgoConnectMock from "../../mocks/myalgowallet-mock";
+
+describe("Webmode - MyAlgo Wallet test cases ", () => {
+	let connector: MyAlgoWalletSession;
+	let sender: Account;
+	let receiver: Account;
+	let algodClient: algosdk.Algodv2;
+	const walletURL: HttpNetworkConfig = {
+		token: "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+		server: "http://localhost",
+		port: 4001,
+	}
+	algodClient = algoexplorerAlgod(walletURL);
+
+	beforeEach(() => {
+		MyAlgoConnect
```

Comment:
what's this line doing?

Reply:
oops I was checking the connection locally, have to remove it, Thanks